### PR TITLE
Fix Github issues - 1909,1908,1768

### PR DESCRIPTION
**Pull request by dewmini**
### Proposed changes in this pull request

This PR consists the fixes for the below mentioned github issues. These issues are basically based on the UI of the Admin web app.  
https://github.com//wso2/product-apim/issues/1909
https://github.com//wso2/product-apim/issues/1908
https://github.com//wso2/product-apim/issues/1768
https://github.com//wso2/product-apim/issues/1737
https://github.com//wso2/product-apim/issues/1770

-

### When should this PR be merged
No pre-conditions.

### Follow up actions

No follow up actions.

-

#### General

- [ ] **Is this PR explained thoroughly?** All code changes must be accounted for in the PR description.
- [ ] **Is the PR labeled correctly?**

#### Functionality

- [ ] **Are all requirements met?** Compare implemented functionality with the requirements specification.
- [ ] **Does the UI work as expected?** There should be no Javascript errors in the console; all resources should load. There should be no unexpected errors. Deliberately try to break the feature to find out if there are corner cases that are not handled.

#### Code

- [ ] **Do you fully understand the introduced changes to the code?** If not ask for clarification, it might uncover ways to solve a problem in a more elegant and efficient way.
- [ ] **Does the PR introduce any inefficient database requests?** Use the debug server to check for duplicate requests.
- [ ] **Are all necessary strings marked for translation?** All strings that are exposed to users via the UI must be [marked for translation](https://docs.djangoproject.com/en/1.10/topics/i18n/translation/).

#### Tests

- [ ] **Are there sufficient test cases?** Ensure that all components are tested individually; models, forms, and serializers should be tested in isolation even if a test for a view covers these components.
- [ ] **If this is a bug fix, are tests for the issue in place?**  There must be a test case for the bug to ensure the issue won’t regress. Make sure that the tests break without the new code to fix the issue.
- [ ] **If this is a new feature or a significant change to an existing feature?** has the manual testing spreadsheet been updated with instructions for manual testing?

#### Security

- [ ] **Confirm this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.**
- [ ] **Are all UI and API inputs run through forms or serializers?**
- [ ] **Are all external inputs validated and sanitized appropriately?**
- [ ] **Does all branching logic have a default case?**
- [ ] **Does this solution handle outliers and edge cases gracefully?**
- [ ] **Are all external communications secured and restricted to SSL?**

#### Documentation

- [ ] **Are changes to the UI documented in the platform docs?** If this PR introduces new platform site functionality or changes existing ones, the changes should be documented.
- [ ] **Are changes to the API documented in the API docs?** If this PR introduces new API functionality or changes existing ones, the changes must be documented.
- [ ] **Are reusable components documented?** If this PR introduces components that are relevant to other developers (for instance a mixin for a view or a generic form) they should be documented in the Wiki.

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin/site/themes/wso2/templates/policy/resource/policy-add/js/policy-designer-new.js [hunk 1]
```diff
@@ -205,6 +205,67 @@ var onExecutionFlowTypeChange = function (id, optionTextOb) {
...
+        return true;
+    }
+};
+
+var onEndIPChange = function (IPtextEle) {
+
+    var validIP = onIPChange(IPtextEle);
+    if(validIP) {
+        var temp = $(IPtextEle)[0].id.split("-");
+        var startIP = $('#ip-range-start-address-input-'+temp[5]);
```
Comment (tgtshanika) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin/site/themes/wso2/templates/policy/resource/policy-add/js/policy-designer-new.js]: Formatting issues in this code segment

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin/site/themes/wso2/templates/policy/resource/policy-add/js/policy-designer-new.js [hunk 2]
```diff
@@ -205,6 +205,67 @@ var onExecutionFlowTypeChange = function (id, optionTextOb) {
...
+
+    if (block > 0 && block < 255) {
+        return true;
+    }
+    return false;
+};
+
+var validIPRange = function (startIP, endIP) {
+
+    var startIPBlocks = startIP.split(".");
```
Comment (tgtshanika) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin/site/themes/wso2/templates/policy/resource/policy-add/js/policy-designer-new.js]: Is there a possibility of becoming the startIP and endIP null? If so, shall we handle null values in here?

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin/site/themes/wso2/templates/policy/resource/policy-add/js/policy-designer-new.js [hunk 3]
```diff
@@ -205,6 +205,67 @@ var onExecutionFlowTypeChange = function (id, optionTextOb) {
...
+var onEndIPChange = function (IPtextEle) {
+
+    var validIP = onIPChange(IPtextEle);
+    if(validIP) {
+        var temp = $(IPtextEle)[0].id.split("-");
+        var startIP = $('#ip-range-start-address-input-'+temp[5]);
+        var validIPRange = validIPRange(startIP,$(IPtextEle).val());
+        var elementId = $(IPtextEle)[0].id;
+        var element = $('#' + elementId);
+        if(!validIPRange){
```
Comment (harsha89) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin/site/themes/wso2/templates/policy/resource/policy-add/js/policy-designer-new.js]: need space before {

**Comment by isurulucky** [comment]
@dewmini Its better if you can include some information on what this PR is about, using the PR template.

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin/modules/user/login.jag [hunk 4]
```diff
@@ -251,6 +251,11 @@ var checkWorkflowServerStatus = function (serverUrl) {
         httpClient.execute(httpPost);
         return true;
     } catch (e) {
+        var errorMessage = e.message;
+        var BPSerror = "java.lang.IllegalArgumentException";
+        if (errorMessage.indexOf(BPSerror) !== -1) {
```
Comment (isurulucky) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin/modules/user/login.jag]: Can errorMessage be null? If so, would need to handle that condition.

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin/site/themes/wso2/templates/policy/resource/policy-add/js/policy-designer-new.js [hunk 5]
```diff
@@ -205,6 +205,67 @@ var onExecutionFlowTypeChange = function (id, optionTextOb) {
...
+        $('#label' + elementId).remove();
+        element.css("border", "1px solid #cccccc");
+        return true;
+    }
+};
+
+var onEndIPChange = function (IPtextEle) {
+
+    var validIP = onIPChange(IPtextEle);
+    if(validIP) {
```
Comment (ChamNDeSilva) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin/site/themes/wso2/templates/policy/resource/policy-add/js/policy-designer-new.js]: Add space between if and ( - formatting

**Pull request merged**